urlpattern/urlpattern.any.js

```diff
@@ -1,2 +1,2 @@
-// META: global=window,worker
+// META: global=window,worker,shadowrealm-in-window,shadowrealm-in-shadowrealm,shadowrealm-in-dedicatedworker,shadowrealm-in-sharedworker
```

Comment:
Why do most of these specify `shadowrealm` but this one specifies this list instead? What is the difference?

Reply:
So these are the subset of `shadowrealm`? Maybe it's better to set `shadowrealm`.
https://github.com/web-platform-tests/wpt/blob/2b8ebc228bb3a083160dd22eabf7812118523230/tools/manifest/sourcefile.py#L100-L106